docs/current_docs/api/clients-sdk.mdx

```diff
@@ -0,0 +1,205 @@
+
+Use the `tree` command to see the build artifacts on the host, as shown below:
+
+```shell
+tree build
+build/
+├── darwin
+│   ├── amd64
+│   │   └── multibuild
+│   └── arm64
+│       └── multibuild
+└── linux
+    ├── amd64
+    │   └── multibuild
+    └── arm64
+        └── multibuild
+```
+</TabItem>
+<TabItem value="Python">
+This example demonstrates how to test a Python application against multiple Python versions using the Python SDK.
+
+Clone an example project:
+
+```shell
+git clone --branch 0.101.0 https://github.com/tiangolo/fastapi
+cd fastapi
+```
+
+Create a new file named `test.py` in the project directory and add the following code to it.
+
+```python file=./snippets/clients-sdk/python/test.py
+```
+
+This Python program imports the Dagger SDK and defines an asynchronous function named `test()`. This `test()` function creates a Dagger client, which provides an interface to the Dagger API. It also defines the test matrix, consisting of Python versions `3.7` to `3.11` and iterates over this matrix, downloading a Python container image for each specified version and testing the source application in that version.
+
+Run the Python program by executing the command below from the project directory:
+
+```shell
+dagger run python test.py
+```
+
+The `dagger run` command executes the specified command in a Dagger session and displays live progress. The tool tests the application against each version concurrently and displays the following final output:
+
+```shell
+Starting tests for Python 3.7
+Tests for Python 3.7 succeeded!
+Starting tests for Python 3.8
+Tests for Python 3.8 succeeded!
+Starting tests for Python 3.9
+Tests for Python 3.9 succeeded!
+Starting tests for Python 3.10
+Tests for Python 3.10 succeeded!
+Starting tests for Python 3.11
+Tests for Python 3.11 succeeded!
+All tasks have finished
+```
+</TabItem>
+<TabItem value="TypeScript">
+
+This example demonstrates how to test a Node.js application against multiple Node.js versions using the TypeScript SDK.
+
+Create an example React project (or use an existing one) in TypeScript:
+
+```shell
+npx create-react-app my-app --template typescript
+cd my-app
+```
+
+In the project directory, create a new file named `build.mts` and add the following code to it:
+
+```typescript file=./snippets/clients-sdk/typescript/build.mts
+```
+
+This TypeScript program imports the Dagger SDK and defines an asynchronous function. This function creates a Dagger client, which provides an interface to the Dagger API. It also defines the test/build matrix, consisting of Node.js versions `12`, `14` and `16`, and iterates over this matrix, downloading a Node.js container image for each specified version and testing and building the source application against that version.
+
+Run the program by executing the command below:
+
+```shell
+dagger run node --loader ts-node/esm ./build.mts
```

Comment:
@TomChv can you please help me update this example? This does not work in Node 20.x.

Reply:
Confirming that it works after installing `ts-node`